### ISSUE TITLE
Even Odd Deriv Bot

### ISSUE THREAD

**Issue by jasonwza** [jasonwza/Even-Odd-Deriv-Bot#1]
I need a Deriv Bot for Even/Odd Trade with these specific rules:

Market: Continuous Indicies: Volatility 100(s) index
Trade Type: Even/Odd
Duration: 5 Ticks
No martingale

Ill explain the rules of Trade in a PDF.

**Comment by holahub23** [comment]
helo can i help you?

**Comment by jasonwza** [comment]
The trade is:

Market: Continuous Indicies: Volatility 100(s) index
Trade Type: Even/Odd
Duration: 5 Ticks
No martingale

We will only take even trade for 5 ticks. 

Basically, we will look for a 5 digit pattern:

We will only look at the last digit of the price. 

Example: 3, 3, 4, 2, 1 - Two odd digits of the same value, followed by two even digits of different value, the 5th digit the trigger for trade. The 5th can be either odd or even. We will initiate an even trade on the 5th digit, if all rules/conditons are correct. 

Check out the attached pdf for detailed info.

[Deriv Even Odd New Bot.pdf](https://github.com/user-attachments/files/17358936/Deriv.Even.Odd.New.Bot.pdf)

**Comment by jasonwza** [comment]
Let me know of any questions.

**Comment by jasonwza** [comment]
Trade is executed on the 5th digit.

**Comment by holahub23** [comment]
Does that mean the 5th number doesn't have to be even, it can be odd too?

**Comment by holahub23** [comment]
and only on Market: Continuous Indicies: Volatility 100(s) index??

**Comment by jasonwza** [comment]
This market (Continuous Indicies: Volatility 100(s) index), is where I tested it the most, but it should work in any Continuous Indicies. But  Volatility 100(s) index is fast moving, hence the bot will execute trade faster, but can use on other Continuous Indicies

**Comment by holahub23** [comment]
Does that mean the 5th number doesn't have to be even, it can be odd too?

**Comment by jasonwza** [comment]
Yes, the 5th digit can be either odd or even. The trade must be executed on the 5th digit. 

The first two digits  must be odd of same value, 3rd and 4th digits must be even of different values and 5th digit can be either odd/even.

**Comment by holahub23** [comment]
when you test it how accurate is this strategy?

**Comment by jasonwza** [comment]
It depends. Because this strategy is only 5 ticks long, a delay in the signal can affect the outcome. But I did manage to get 75 to 85% on multiple testings.

**Comment by jasonwza** [comment]
A bot might increase the win rate
